infrastructure/template.yaml

```diff
@@ -348,8 +348,11 @@ Resources:
         IncludeExecutionData: True
         Level: ALL
       Policies:
-        - AWSSecretsManagerGetSecretValuePolicy:
-            SecretArn: !Sub arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:HMRC/BearerToken/*
```

Comment:
I usually prefer this, than the longer form

Reply:
That didn't work for some reason